### PR TITLE
[AL-15] Create maunal register api

### PR DESCRIPTION
**Pull request by pvnamGit**
## Jira link
[AL-15](https://capstone-alumni.atlassian.net/jira/software/c/projects/AL/boards/1?modal=detail&selectedIssue=AL-15)

## Describe your changes
- Implemented basic api for creating new alum
- Not handle role yet

## Postman (BE)
Screenshot from 2022-12-26 16-52-05
Screenshot from 2022-12-26 17-05-10

## Checklist before requesting a review
1. [x] I have performed a self-review of my code
2. [x] Have you lint your code locally before submission?

### CODE REVIEW

File: prisma/schema.prisma [hunk 1]
```diff
@@ -46,9 +46,43 @@ model User {
...
+  roles         UserRole[]
+  archived      Boolean   @default(false)
+  createdAt     DateTime  @default(now()) @map("created_at")
+  updatedAt     DateTime? @map("updated_at")
+  accessStatus  AccessStatus  @default(PENDING) @map("access_status")
 
   @@map("users")
 }
+
+model Role {
```
Comment (pvnamGit) [inline prisma/schema.prisma:63]: nevermind about the role here, i just implemented to test role-based authorize but forgot to revert. I will refactor it in task AL-32

File: lib/prisma/alumni.ts [hunk 2]
```diff
@@ -0,0 +1,61 @@
+import { prisma } from '@/lib/prisma/prisma';
+import { AlumCreateRequest } from 'pages/types/apiRequests';
+import { isNull } from 'lodash';
+import { exclude } from './helpers';
```
Comment (congtrung6391) [inline lib/prisma/alumni.ts:4]: could you try omit of lodash

Comment (pvnamGit) [reply lib/prisma/alumni.ts:4]: I followed guide from [prisma](https://www.prisma.io/docs/concepts/components/prisma-client/excluding-fields#excluding-the-password-field). It LGTM

Comment (congtrung6391) [reply lib/prisma/alumni.ts:4]: https://lodash.com/docs/4.17.15#omit
It's totally the same :v

**Review by congtrung6391 — APPROVED** [review]

**Pull request merged**